JavaScriptSPA/policies.js

```diff
@@ -0,0 +1,5 @@
+
+// add here different b2c policies you have
+const b2cPolicies = {
+    signUpSignIn: "b2c_1_susi", // Enter here the policy name string you created on the portal
```

Comment:
Are we planning to add different policies later?

Reply:
might be a good way to organize strings if the user wants to try out different policies to compare. what would you suggest?

Reply:
Requiring the user to edit another file to get it working with their tenant adds to the complexity of steps in the tutorial.

Having constants in another object is certainly something that might be done in a real-world app, but in a day-1 learning situation, it's better to present more literal code. In this case, I'd like to keep the URI with the policy name intact as opposed to concatenation.

In these types of tutorials, to quote Dave Cheney, [clear is better than clever](https://dave.cheney.net/2019/07/09/clear-is-better-than-clever).

Reply:
sounds good @mmacy